sources/web/datalab/polymer/components/auth-panel/auth-panel.ts

```diff
@@ -19,8 +19,8 @@
  */
 class AuthPanel extends Polymer.Element {
 
-  private _signedIn: boolean;
-  private _userInfo: string;
+  protected _signedIn: boolean;
```

Comment:
The right fix here would be to remove these modifiers altogether, effectively making these public properties, which they actually are, since they're being accessed from the HTML. We were abusing the fact that typescript used to not flagging write-only accessed variables to treat them as private, but they're really not.

We could still name them with an underscore though, to signal that they're really just private to this web component and shouldn't be accessed directly from outside the component.

Reply:
Additionally, can we please separate this cleanup from the actual change? It's not a small cleanup.